Assets/MixedRealityToolkit/_Core/Utilities/Lines/DataProviders/SimpleLineDataProvider.cs

```diff
@@ -74,6 +74,9 @@ protected override void SetPointInternal(int pointIndex, Vector3 point)
         {
             switch (pointIndex)
             {
+                case 0:
+                    endPoint.Position = point;
```

Comment:
Should this be `startPoint.Position` to match https://github.com/StephenHodgson/MixedRealityToolkit-Unity/blob/c2e196261d24ed8774589ba3b073b963e4ab2388/Assets/MixedRealityToolkit/_Core/Utilities/Lines/DataProviders/SimpleLineDataProvider.cs#L63?

Reply:
This fixes a bug.

Reply:
What was the bug? Why would `case 0` set one thing (`endPoint`) but get a different thing (`startPoint`)?

Reply:
Because we need to set the first point, and we weren't

Reply:
I don't understand what you're getting at

Reply:
Oh I see, this should be setting the start point. haha. (why didn't you just say that?) 😆

Reply:
> (why didn't you just say that?)

>Should this be **startPoint.Position** to match https://github.com/StephenHodgson/MixedRealityToolkit-Unity/blob/c2e196261d24ed8774589ba3b073b963e4ab2388/Assets/MixedRealityToolkit/_Core/Utilities/Lines/DataProviders/SimpleLineDataProvider.cs#L63?

🙃

Reply:
😆 Wow all I saw was the link

Reply:
Fixed